src/connectors/h1_collect.py

```diff
@@ -92,17 +109,8 @@ def load_data(
     """Perform the API call"""
     headers: dict = {"Accept": "application/json"}
 
-    try:
-        log.debug(f"Preparing GET: url={url} with params={params}")
-        resp = requests.get(
-            url, auth=(api_identifier, token), params=params, headers=headers
-        )
-        resp.raise_for_status()
-    except HTTPError as http_err:
-        log.error(f"HTTP error occurred: {http_err}")
-        raise
     log.debug(resp.status_code)
-    return resp
```

Comment:
you'll need to either skip the type declaration on the function or update it to match the return type. you can either set up mypy to run locally, or see ci/cd results [here](https://app.circleci.com/pipelines/github/snowflakedb/SnowAlert?branch=fix_amount_and_transactions)